src/main/java/ui/components/pickers/AssigneePickerState.java

```diff
@@ -0,0 +1,82 @@
+package ui.components.pickers;
+
+import util.Utility;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class AssigneePickerState {
+    private List<PickerAssignee> currentAssigneesList;
```

Comment:
I think the better name is `...UsersList` or `...MembersList`? There can be only one assignee but there can be a list of users. Create a separate issue to change Assignee to a more appropriate name (Member/User) across all this code.